project/Dependencies.scala

```diff
@@ -2,7 +2,7 @@ import sbt._
 
 object Dependencies {
   lazy val lenthallV = "0.20"
-  lazy val wdl4sV = "0.8"
+  lazy val wdl4sV = "0.9-796cfb6-SNAP"
```

Comment:
Leaving a note to make sure some non-snapshot version of wdl4s is used before merging into a release branch. Anything with `-SNAP` is eligible to be cleaned up / deleted from artifactory in the future, and the "stable" hotfix build of cromwell will break.